src/ucp/proto/proto_common.inl

```diff
@@ -255,7 +258,7 @@ ucp_proto_request_send_op(ucp_ep_h ep, ucp_proto_select_t *proto_select,
 static UCS_F_ALWAYS_INLINE size_t
 ucp_proto_request_pack_rkey(ucp_request_t *req, uint64_t distance_dev_map,
                             const ucs_sys_dev_distance_t *dev_distance,
-                            void *rkey_buffer)
+                            void *rkey_buffer, ucp_md_map_t md_map)
```

Comment:
i guess it's needed because we need to send less md's than may be registered by existing memh?
maybe assert that md_map is contained in memh->md_map?